platforms/tktrex/backend/package.json

```diff
@@ -7,10 +7,10 @@
     "lint": "eslint ./bin ./lib ./models ./test",
     "test": "jest",
     "coc": "DEBUG=* ts-node bin/count-o-clock.js",
-    "watch": "key=fuffa DEBUG=\"*,-express:*,-body-parser:*,-send\" ts-node-dev -r tsconfig-paths/register --respawn --transpile-only bin/server",
```

Comment:
perhaps we can also get rid of all the environment variables and use `.env.development` as standard practice?

Reply:
I have to plan this, cause I may need to touch also the `config/settings.json` to avoid duplicated variables and the impact can be huge.